configs/eslint-config/eslint.config.mjs

```diff
@@ -47,7 +47,7 @@ export default [
       '@typescript-eslint/no-unsafe-member-access': 'warn',
       '@typescript-eslint/no-floating-promises': 'warn',
       '@typescript-eslint/no-unsafe-call': 'warn',
-      '@typescript-eslint/no-throw-literal': 'warn',
+      '@typescript-eslint/only-throw-error': 'warn',
```

Comment:
The @typescript-eslint plugin does not have a no-throw-literal rule. Instead, I replaced it with @typescript-eslint/only-throw-error, which enforces the same rule.